### PR TITLE
Add + Delete Product

### PR DESCRIPTION
**Pull request by ngocnm-0258**
## Related Tickets:
- [#66634](https://edu-redmine.sun-asterisk.vn/issues/66634)
- [#66363](https://edu-redmine.sun-asterisk.vn/issues/66363)

## WHAT this PR does?
- Add Product + Delete product

## Notes Impacted Areas:
- N/A.

## Attachments:
image
image

### CODE REVIEW

**Comment by ngocnm-0258** [comment]
ready

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -26,22 +27,48 @@ public function index()
...
+            $image->move(public_path('images'), $imageName);
+            $imagePath = env('IMAGE_PATH');
+            $product->photo = $imagePath . $imageName;
+        }
+
+        $product->save();
+
+        $products = DB::table('products')->where('salesman_id', $product->salesman_id)
+            ->orderBy('id', 'desc')
+            ->paginate('8');
```
Comment (ghost) [inline app/Http/Controllers/ProductController.php]: paginate => config

**Comment by ngocnm-0258** [comment]
ready

**Comment by ghost** [comment]
chua fix comment

**Comment by ghost** [comment]
commented

**Comment by ngocnm-0258** [comment]
em nhầm sửa function dưới mà sót function trên ạ :((

**Comment by ngocnm-0258** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**